components/admin/permit-holders/table/ConfirmSetInactiveModal.tsx

```diff
@@ -11,11 +11,20 @@ import {
   FormControl,
   FormLabel,
   Textarea,
+  useToast,
 } from '@chakra-ui/react'; // Chakra UI
+import { useMutation } from '@apollo/client';
+import {
+  SetApplicantAsInactiveRequest,
+  SetApplicantAsInactiveResponse,
+  SET_APPLICANT_AS_INACTIVE,
+} from '@tools/admin/permit-holders/permit-holders-table';
 
 //Props
 type SetPermitHolderToInactiveModalProps = {
   readonly isOpen: boolean;
+  readonly permitHolderId: number;
```

Comment:
```suggestion
  readonly applicantId: number;
```
We typically use `applicantId`